src/components/Orders.js

```diff
@@ -27,6 +27,7 @@ class Orders extends Component {
 
   componentWillMount() {
     this.props.loadOrders();
+    this.props.loadItems();
```

Comment:
am calling loadItems when we get to the orders page. there should probably be a call to get all the initial states - master items, orders, etc.